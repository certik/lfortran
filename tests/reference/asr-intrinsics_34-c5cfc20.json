{
    "basename": "asr-intrinsics_34-c5cfc20",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_34.f90",
    "infile_hash": "44268c572f1e8d1a0fd2e6f429d0b261bbf840fc385ec8100a7c5a96",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_34-c5cfc20.stdout",
    "stdout_hash": "27acf4e3008beb04e2b1be9b80fd5e1dacf1ef578c05c2f5f9522db7",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}