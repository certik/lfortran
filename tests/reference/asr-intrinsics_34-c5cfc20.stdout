(TranslationUnit (SymbolTable 1 {intrinsics_34: (Program (SymbolTable 2 {epsilon: (ExternalSymbol 2 epsilon 4 epsilon lfortran_intrinsic_math [] epsilon Private), epsilon@depsilon: (ExternalSymbol 2 epsilon@depsilon 4 depsilon lfortran_intrinsic_math [] depsilon Private), epsilon@sepsilon: (ExternalSymbol 2 epsilon@sepsilon 4 sepsilon lfortran_intrinsic_math [] sepsilon Private), x: (Variable 2 x Local (RealConstant   3.14299999999999979e+00 (Real 4 [])) () Save (Real 4 []) Source Public Required .false.), y: (Variable 2 y Local (Cast (RealConstant   2.33000000000000007e+00 (Real 4 [])) RealToReal (Real 8 []) (RealConstant   2.33000000000000007e+00 (Real 8 []))) () Save (Real 8 []) Source Public Required .false.)}) intrinsics_34 [] [(Print () [(FunctionCall 2 epsilon@sepsilon 2 epsilon [((Var 2 x))] (Real 4 []) (RealConstant   1.17549435082228751e-38 (Real 4 [])) ())] () ()) (Print () [(FunctionCall 2 epsilon@depsilon 2 epsilon [((Var 2 y))] (Real 8 []) (RealConstant  2.22507385850720138e-308 (Real 8 [])) ())] () ())]), iso_c_binding: (IntrinsicModule lfortran_intrinsic_iso_c_binding), iso_fortran_env: (IntrinsicModule lfortran_intrinsic_iso_fortran_env), lfortran_intrinsic_builtin: (IntrinsicModule lfortran_intrinsic_builtin), lfortran_intrinsic_math: (IntrinsicModule lfortran_intrinsic_math)}) [])
